ignite/cmd/scaffold.go

```diff
@@ -25,6 +25,28 @@ const (
 	msgCommitPrompt = "Do you want to proceed without committing your saved changes"
 
 	statusScaffolding = "Scaffolding..."
+
+	supportFieldTypes = `
+Currently supports: 
+
+| Type         | Alias   | Index | Code Type | Description                     |
+|--------------|---------|-------|-----------|---------------------------------|
+| string       | -       | yes   | string    | Text type                       |
+| array.string | strings | no    | []string  | List of text type               |
+| bool         | -       | yes   | bool      | Boolean type                    |
+| int          | -       | yes   | int32     | Integer type                    |
+| array.int    | ints    | no    | []int32   | List of integers types          |
+| uint         | -       | yes   | uint64    | Unsigned integer type           |
+| array.uint   | uints   | no    | []uint64  | List of unsigned integers types |
+| coin         | -       | no    | sdk.Coin  | Cosmos SDK coin type            |
+| array.coin   | coins   | no    | sdk.Coins | List of Cosmos SDK coin types   |
+
+Field Usage:
+    - fieldName
+    - fieldName:fieldType
+
+If no :fieldType, default (string) is used
+`
```

Comment:
Maybe this can be part of the `type` command's long description instead of a variable because it's only referenced there, what do you think?

Reply:
I actually think it's a good idea to keep the table in a variable because we would be able to use it in `scaffold list/map/single/query/message` directly, without having to say "For a list of supported types see `ignite scaffold type --help`".